### ISSUE TITLE
Mullvad servers need updating

### ISSUE THREAD

**Issue by pdobrien** [haugene/vpn-configs-contrib#167]
### Do you have the latest provider files?

- [X] I have checked that the provider are the latest

### Have you tested the provider files?

- [X] I have tested the provider files

### Can you create a PR for this config?

- [ ] I can create a PR for this

### Provider details

Mullvad servers need updating again.  For example, us-rag servers used to be us-rag-<number>.relays.mullvad.net which no longer exist.

The us-all file contains more servers than are supported by ovpn which causes an error when used.

Some server lists are also missing, for example us-qas.

It must be possible to programmatically generate the configs from the content at https://mullvad.net/en/servers/ ?

**Comment by scuffe82** [comment]
It looks like they dropped the DNS records for the relays. The names and IP's are the same still but you can't resolve them by name currently. Also the ports changed.

![image](https://user-images.githubusercontent.com/987404/208963053-4a731fe6-2d84-416d-a894-a65d37abcb7e.png)

**Comment by pkishino** [comment]
Feel free to check who last submitted configs to mullvad and tag them here, perhaps they are willing to help you out or else look into creating a PR and contributing.
Thanks

**Comment by scuffe82** [comment]
I was able to get it working by switching to "custom" for the provider. Just create a default.ovpn using the mullvad config from the repo in the config directory that you map to /etc/openvpn/custom and change the server names to the ones you want to use.

docker run --cap-add=NET_ADMIN -d \
-v /myshare:/data \
-v /myshare/config:/etc/openvpn/custom \
-e OPENVPN_PROVIDER=custom \
-e OPENVPN_USERNAME=MULLVADID \
-e OPENVPN_PASSWORD=m \
-e LOCAL_NETWORK=MYLOCALNETWORK \
--log-driver json-file \
--log-opt max-size=10m \
-p 9091:9091 \
--sysctl net.ipv6.conf.all.disable_ipv6=0 \
haugene/transmission-openvpn

**Comment by dak108** [comment]
Additionally the ca_all and ca_mtr require updating. Attached are the config files with the updated hosts.
[ca_mtr.txt](https://github.com/haugene/vpn-configs-contrib/files/10328231/ca_mtr.txt)
[ca_all.txt](https://github.com/haugene/vpn-configs-contrib/files/10328232/ca_all.txt)

**Comment by dak108** [comment]
PR requests #175 #176 to update the Canadian mullvad servers. Montreal servers were the only ones requiring changes

**Comment by edge360** [comment]
I wasn't sure if I should open a new issue, but I was having some trouble using Mullvad - nyc ovpn servers in particular. After looking into it further only a few were working requiring a few restarts before it found a working one. Look like some of naming scheme has changed? us-nyc-ovpn-201.relays.mullvad.net VS us-nyc-201.relays.mullvad.net

 I tested the US servers and only about half of them could be resolved. Posted below.

I wasn't sure how the ovpn files are generated, whether programmatically or manually entered, but the servers are available via Mullvad API - https://api.mullvad.net/www/relays/openvpn/



`host us-atl-ovpn-001.relays.mullvad.net
us-atl-ovpn-001.relays.mullvad.net has address 45.134.140.156
us-atl-ovpn-001.relays.mullvad.net has IPv6 address 2a02:6ea0:c122:3::1f

host us-atl-ovpn-002.relays.mullvad.net
us-atl-ovpn-002.relays.mullvad.net has address 45.134.140.169
us-atl-ovpn-002.relays.mullvad.net has IPv6 address 2a02:6ea0:c122:4::2f

host us-atl-ovpn-101.relays.mullvad.net
us-atl-ovpn-101.relays.mullvad.net has address 66.115.180.226
us-atl-ovpn-101.relays.mullvad.net has IPv6 address 2607:f7a0:1:d::1f

host us-atl-ovpn-102.relays.mullvad.net
us-atl-ovpn-102.relays.mullvad.net has address 66.115.180.227
us-atl-ovpn-102.relays.mullvad.net has IPv6 address 2607:f7a0:1:d::2f

host us-atl-ovpn-103.relays.mullvad.net
us-atl-ovpn-103.relays.mullvad.net has address 66.115.180.228
us-atl-ovpn-103.relays.mullvad.net has IPv6 address 2607:f7a0:1:d::3f

host us-atl-ovpn-104.relays.mullvad.net
us-atl-ovpn-104.relays.mullvad.net has address 66.115.180.229
us-atl-ovpn-104.relays.mullvad.net has IPv6 address 2607:f7a0:1:d::4f

host us-atl-ovpn-105.relays.mullvad.net
us-atl-ovpn-105.relays.mullvad.net has address 66.115.180.230
us-atl-ovpn-105.relays.mullvad.net has IPv6 address 2607:f7a0:1:d::5f

host us-chi-001.relays.mullvad.net
us-chi-001.relays.mullvad.net has address 68.235.43.122

host us-chi-002.relays.mullvad.net
us-chi-002.relays.mullvad.net has address 68.235.43.10

host us-chi-003.relays.mullvad.net
us-chi-003.relays.mullvad.net has address 68.235.43.18

host us-chi-004.relays.mullvad.net
us-chi-004.relays.mullvad.net has address 68.235.43.26

host us-chi-005.relays.mullvad.net
us-chi-005.relays.mullvad.net has address 68.235.43.34

host us-dal-201.relays.mullvad.net
us-dal-201.relays.mullvad.net not found: 3(NXDOMAIN)

host us-dal-202.relays.mullvad.net
us-dal-202.relays.mullvad.net not found: 3(NXDOMAIN)

host us-dal-203.relays.mullvad.net
us-dal-203.relays.mullvad.net not found: 3(NXDOMAIN)

host us-dal-204.relays.mullvad.net
us-dal-204.relays.mullvad.net has address 193.27.13.66

host us-dal-ovpn-101.relays.mullvad.net
us-dal-ovpn-101.relays.mullvad.net has address 174.127.113.3
us-dal-ovpn-101.relays.mullvad.net has IPv6 address 2606:2e00:8007:1::1f

host us-dal-ovpn-102.relays.mullvad.net
us-dal-ovpn-102.relays.mullvad.net has address 174.127.113.4
us-dal-ovpn-102.relays.mullvad.net has IPv6 address 2606:2e00:8007:1::2f

host us-dal-ovpn-103.relays.mullvad.net
us-dal-ovpn-103.relays.mullvad.net has address 174.127.113.5
us-dal-ovpn-103.relays.mullvad.net has IPv6 address 2606:2e00:8007:1::3f

host us-dal-ovpn-104.relays.mullvad.net
us-dal-ovpn-104.relays.mullvad.net has address 174.127.113.6
us-dal-ovpn-104.relays.mullvad.net has IPv6 address 2606:2e00:8007:1::4f

host us-dal-ovpn-105.relays.mullvad.net
us-dal-ovpn-105.relays.mullvad.net has address 174.127.113.7
us-dal-ovpn-105.relays.mullvad.net has IPv6 address 2606:2e00:8007:1::5f

host us-den-ovpn-001.relays.mullvad.net
us-den-ovpn-001.relays.mullvad.net has address 198.54.128.74
us-den-ovpn-001.relays.mullvad.net has IPv6 address 2607:9000:2000:18::1f

host us-den-ovpn-002.relays.mullvad.net
us-den-ovpn-002.relays.mullvad.net has address 198.54.128.66
us-den-ovpn-002.relays.mullvad.net has IPv6 address 2607:9000:2000:17::2f

host us-den-ovpn-003.relays.mullvad.net
us-den-ovpn-003.relays.mullvad.net has address 198.54.128.106
us-den-ovpn-003.relays.mullvad.net has IPv6 address 2607:9000:2000:22::3f

host us-lax-001.relays.mullvad.net
us-lax-001.relays.mullvad.net has address 89.46.114.15

host us-lax-002.relays.mullvad.net
us-lax-002.relays.mullvad.net not found: 3(NXDOMAIN)

host us-lax-003.relays.mullvad.net
us-lax-003.relays.mullvad.net not found: 3(NXDOMAIN)

host us-lax-004.relays.mullvad.net
us-lax-004.relays.mullvad.net not found: 3(NXDOMAIN)

host us-lax-005.relays.mullvad.net
us-lax-005.relays.mullvad.net not found: 3(NXDOMAIN)

host us-lax-006.relays.mullvad.net
us-lax-006.relays.mullvad.net not found: 3(NXDOMAIN)

host us-lax-007.relays.mullvad.net
us-lax-007.relays.mullvad.net not found: 3(NXDOMAIN)

host us-lax-008.relays.mullvad.net
us-lax-008.relays.mullvad.net not found: 3(NXDOMAIN)

host us-lax-009.relays.mullvad.net
us-lax-009.relays.mullvad.net not found: 3(NXDOMAIN)

host us-lax-010.relays.mullvad.net
us-lax-010.relays.mullvad.net not found: 3(NXDOMAIN)

host us-lax-011.relays.mullvad.net
us-lax-011.relays.mullvad.net not found: 3(NXDOMAIN)

host us-lax-012.relays.mullvad.net
us-lax-012.relays.mullvad.net not found: 3(NXDOMAIN)

host us-lax-013.relays.mullvad.net
us-lax-013.relays.mullvad.net not found: 3(NXDOMAIN)

host us-lax-014.relays.mullvad.net
us-lax-014.relays.mullvad.net not found: 3(NXDOMAIN)

host us-lax-ovpn-101.relays.mullvad.net
us-lax-ovpn-101.relays.mullvad.net has address 198.54.129.82
us-lax-ovpn-101.relays.mullvad.net has IPv6 address 2607:9000:3000:18::1f

host us-lax-ovpn-102.relays.mullvad.net
us-lax-ovpn-102.relays.mullvad.net has address 198.54.129.74
us-lax-ovpn-102.relays.mullvad.net has IPv6 address 2607:9000:3000:17::2f

host us-lax-ovpn-201.relays.mullvad.net
us-lax-ovpn-201.relays.mullvad.net has address 169.150.203.41

host us-lax-ovpn-202.relays.mullvad.net
us-lax-ovpn-202.relays.mullvad.net has address 169.150.203.54

host us-mia-201.relays.mullvad.net
us-mia-201.relays.mullvad.net not found: 3(NXDOMAIN)

host us-mia-202.relays.mullvad.net
us-mia-202.relays.mullvad.net not found: 3(NXDOMAIN)

host us-mia-203.relays.mullvad.net
us-mia-203.relays.mullvad.net not found: 3(NXDOMAIN)

host us-mia-204.relays.mullvad.net
us-mia-204.relays.mullvad.net not found: 3(NXDOMAIN)

host us-mia-205.relays.mullvad.net
us-mia-205.relays.mullvad.net not found: 3(NXDOMAIN)

host us-mia-206.relays.mullvad.net
us-mia-206.relays.mullvad.net not found: 3(NXDOMAIN)

host us-nyc-201.relays.mullvad.net
us-nyc-201.relays.mullvad.net not found: 3(NXDOMAIN)

host us-nyc-202.relays.mullvad.net
us-nyc-202.relays.mullvad.net not found: 3(NXDOMAIN)

host us-nyc-203.relays.mullvad.net
us-nyc-203.relays.mullvad.net not found: 3(NXDOMAIN)

host us-nyc-204.relays.mullvad.net
us-nyc-204.relays.mullvad.net not found: 3(NXDOMAIN)

host us-nyc-205.relays.mullvad.net
us-nyc-205.relays.mullvad.net not found: 3(NXDOMAIN)

host us-nyc-206.relays.mullvad.net
us-nyc-206.relays.mullvad.net not found: 3(NXDOMAIN)

host us-nyc-207.relays.mullvad.net
us-nyc-207.relays.mullvad.net has address 89.46.62.93

host us-nyc-208.relays.mullvad.net
us-nyc-208.relays.mullvad.net not found: 3(NXDOMAIN)

host us-nyc-209.relays.mullvad.net
us-nyc-209.relays.mullvad.net not found: 3(NXDOMAIN)

host us-nyc-211.relays.mullvad.net
us-nyc-211.relays.mullvad.net not found: 3(NXDOMAIN)

host us-nyc-212.relays.mullvad.net
us-nyc-212.relays.mullvad.net not found: 3(NXDOMAIN)

host us-nyc-213.relays.mullvad.net
us-nyc-213.relays.mullvad.net not found: 3(NXDOMAIN)

host us-nyc-214.relays.mullvad.net
us-nyc-214.relays.mullvad.net not found: 3(NXDOMAIN)

host us-nyc-215.relays.mullvad.net
us-nyc-215.relays.mullvad.net not found: 3(NXDOMAIN)

host us-nyc-216.relays.mullvad.net
us-nyc-216.relays.mullvad.net not found: 3(NXDOMAIN)

host us-nyc-217.relays.mullvad.net
us-nyc-217.relays.mullvad.net not found: 3(NXDOMAIN)

host us-nyc-401.relays.mullvad.net
us-nyc-401.relays.mullvad.net has address 198.44.136.130

host us-nyc-402.relays.mullvad.net
us-nyc-402.relays.mullvad.net has address 198.44.136.162

host us-nyc-403.relays.mullvad.net
us-nyc-403.relays.mullvad.net has address 198.44.136.194

host us-phx-ovpn-101.relays.mullvad.net
us-phx-ovpn-101.relays.mullvad.net has address 198.54.133.34
us-phx-ovpn-101.relays.mullvad.net has IPv6 address 2607:9000:7000:12::101f

host us-phx-ovpn-102.relays.mullvad.net
us-phx-ovpn-102.relays.mullvad.net has address 198.54.133.50
us-phx-ovpn-102.relays.mullvad.net has IPv6 address 2607:9000:7000:13::102f

host us-phx-ovpn-103.relays.mullvad.net
us-phx-ovpn-103.relays.mullvad.net has address 198.54.133.66
us-phx-ovpn-103.relays.mullvad.net has IPv6 address 2607:9000:7000:14::103f

host us-qas-001.relays.mullvad.net
us-qas-001.relays.mullvad.net has address 198.54.135.162

host us-qas-002.relays.mullvad.net
us-qas-002.relays.mullvad.net has address 198.54.135.194

host us-qas-ovpn-101.relays.mullvad.net
us-qas-ovpn-101.relays.mullvad.net has address 185.156.46.169
us-qas-ovpn-101.relays.mullvad.net has IPv6 address 2a02:6ea0:e206:4::a01f

host us-qas-ovpn-102.relays.mullvad.net
us-qas-ovpn-102.relays.mullvad.net has address 185.156.46.182
us-qas-ovpn-102.relays.mullvad.net has IPv6 address 2a02:6ea0:e206:5::a02f

host us-rag-ovpn-101.relays.mullvad.net
us-rag-ovpn-101.relays.mullvad.net has address 198.54.130.34
us-rag-ovpn-101.relays.mullvad.net has IPv6 address 2607:9000:4000:12::101f

host us-rag-ovpn-102.relays.mullvad.net
us-rag-ovpn-102.relays.mullvad.net has address 198.54.130.50
us-rag-ovpn-102.relays.mullvad.net has IPv6 address 2607:9000:4000:13::102f

host us-rag-ovpn-103.relays.mullvad.net
us-rag-ovpn-103.relays.mullvad.net has address 198.54.130.66
us-rag-ovpn-103.relays.mullvad.net has IPv6 address 2607:9000:4000:14::103f

host us-sea-ovpn-101.relays.mullvad.net
us-sea-ovpn-101.relays.mullvad.net has address 198.54.131.34
us-sea-ovpn-101.relays.mullvad.net has IPv6 address 2607:9000:5000:12::101f

host us-sea-ovpn-102.relays.mullvad.net
us-sea-ovpn-102.relays.mullvad.net has address 198.54.131.50
us-sea-ovpn-102.relays.mullvad.net has IPv6 address 2607:9000:5000:13::102f

host us-sea-ovpn-103.relays.mullvad.net
us-sea-ovpn-103.relays.mullvad.net has address 198.54.131.66
us-sea-ovpn-103.relays.mullvad.net has IPv6 address 2607:9000:5000:14::103f

host us-sjc-ovpn-001.relays.mullvad.net
us-sjc-ovpn-001.relays.mullvad.net has address 198.54.134.34
us-sjc-ovpn-001.relays.mullvad.net has IPv6 address 2607:9000:8000:12::1f

host us-sjc-ovpn-002.relays.mullvad.net
us-sjc-ovpn-002.relays.mullvad.net has address 198.54.134.66
us-sjc-ovpn-002.relays.mullvad.net has IPv6 address 2607:9000:8000:13::2f

host us-sjc-ovpn-003.relays.mullvad.net
us-sjc-ovpn-003.relays.mullvad.net not found: 3(NXDOMAIN)

host us-slc-ovpn-101.relays.mullvad.net
us-slc-ovpn-101.relays.mullvad.net has address 69.4.234.132
us-slc-ovpn-101.relays.mullvad.net has IPv6 address 2606:2e00:0:b9::1f

host us-slc-ovpn-102.relays.mullvad.net
us-slc-ovpn-102.relays.mullvad.net has address 69.4.234.133
us-slc-ovpn-102.relays.mullvad.net has IPv6 address 2606:2e00:0:b9::2f

host us-slc-ovpn-103.relays.mullvad.net
us-slc-ovpn-103.relays.mullvad.net has address 69.4.234.134
us-slc-ovpn-103.relays.mullvad.net has IPv6 address 2606:2e00:0:b9::3f

host us-slc-ovpn-104.relays.mullvad.net
us-slc-ovpn-104.relays.mullvad.net has address 69.4.234.135
us-slc-ovpn-104.relays.mullvad.net has IPv6 address 2606:2e00:0:b9::4f

host us-slc-ovpn-105.relays.mullvad.net
us-slc-ovpn-105.relays.mullvad.net has address 69.4.234.136
us-slc-ovpn-105.relays.mullvad.net has IPv6 address 2606:2e00:0:b9::5f

host us-slc-ovpn-106.relays.mullvad.net
us-slc-ovpn-106.relays.mullvad.net has address 69.4.234.137
us-slc-ovpn-106.relays.mullvad.net has IPv6 address 2606:2e00:0:b9::6f
`

**Comment by wickedyoda** [comment]
I just came looking for the same thing; the container will not spin up and connect; it's looking for default.ovpn. When loading, it pulls the configs, and when it cant find it, it restarts. Been running this container with Mull for almost 3 years and only since this update was pushed that Ive had issues.  I will load github and look at them locally to see if I can help.

**Comment by Downy** [comment]
Can confirm, this update broke the container.
Adding the ```OPENVPN_CONFIG=xx_xxx``` setting to the docker compose let startup progress a little further but it will now hang on ```Initalization Sequence Completed``` and never actually start Transmission

**Comment by wickedyoda** [comment]
Yes, you cant leave it on default anymore. If you want it to work, then you have to specify the connection location/profile in the docker-compose now. For the last 2 years, mines been at default setting; if I hadn't have noticed mine not running when I went to add a download, I would not have realized it was broken and went investigating. 

When you specify the profile you also cant leave the .ovpn on the end of the profile name. 

@C9Glax

**Comment by C9Glax** [comment]
Thanks for pinging me @wickedyoda 
#191 should fix this.

> When you specify the profile you also cant leave the .ovpn on the end of the profile name.

That is always the case in this container.

> `Initalization Sequence Completed` and never actually start Transmission

Should actually not have anything to do with this...

**Comment by C9Glax** [comment]
Just a quick word of advice for everyone: Don't use default config.
You are crippling your speeds by choosing a random global server.
Use either
a) A server local to you (country or city)
b) A server in a country that has lax laws regarding torrenting (e.g. not germany)

**Comment by wickedyoda** [comment]
I had my container set to reboot every so many hour so the connection location would always change; now, the only way to do so is to edit the config in docker-compose. Do you have any other advice?

**Comment by C9Glax** [comment]
No, it's fixed now. default.ovpn is back. just revert to old setting.

**Comment by bitsandbooks** [comment]
Sorry to bug, I guess I'm a bit confused what to do here. I'm specifying the location in my config -- in fact, I was even before this -- but I still can't connect. What am I doing wrong?

**Comment by C9Glax** [comment]
You aren't bothering, and in fact it was a problem I created. Once #192 is merged, it should work again (have a look here: #194 )
